frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -587,6 +588,12 @@ export const insightLogic = kea<insightLogicType>([
                 saveInsightFailure: () => false,
             },
         ],
+        acknowledgedRefreshButtonChanged: [
+            !!localStorage.getItem('acknowledged_refresh_button_changed') as boolean,
```

Comment:
We're already using `kea-localstorage`, so no need to access `localStorage` in this file – just pass `{ persist: true }` as the second argument of a reducer. Search the codebase for examples.
BTW also no need for a loader here, there's no async logic, so a plain reducer is enough.

Reply:
Wait what loader?

Reply:
My bad, for some reason GitHub's code folding thinks this is under `loaders` rather than `reducers` 😄 
<img width="663" alt="Screenshot 2023-02-13 at 17 18 09" src="https://user-images.githubusercontent.com/4550621/218512589-f34e8faf-d89c-4b79-aaf0-e70ff7b2d59e.png">

Reply:
ah I see

I think the loaders are grouped in as the lines under the change above